packages/mail/src/mail.spec.js

```diff
@@ -48,4 +50,10 @@ describe('sgMail.send()', () => {
         expect(response.statusCode).to.equal(201);
       });
   });
+
+
+  it('should throw an error if callback is not a function', () => {
+    return expect(sgMail.send(data, false, cb)).to.be.an('error');
```

Comment:
This test will need to be adjusted so that it expects a thrown error. You can do this as such:

```js
expect(function() {
  sgMail.send(data, false, {});
}).to.throw(Error);
```

Reply:
Fixed..